src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -428,8 +467,8 @@ public void handlesRequestInTime() throws IOException {
     @Test
     public void notDistortContent() throws IOException {
         final int length = 1000000;
-        final File temp = File.createTempFile("notDistortContent", ".tmp");
-        final BufferedWriter bwr = new BufferedWriter(new FileWriter(temp));
+        final File tempfile = this.temp.newFile("notDistortContent.tmp");
```

Comment:
@raphaelln it's not about removing `this`, please read http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html